src/Surfnet/StepupRa/RaBundle/Command/ExportRaSecondFactorsCommand.php

```diff
@@ -88,18 +93,19 @@
      * @param string $institution
      * @return ExportRaSecondFactorsCommand
      */
-    public static function fromSearchCommand(SearchRaSecondFactorsCommand $command, $institution)
+    public static function fromSearchCommand(SearchRaSecondFactorsCommand $command, $actorInstitution)
```

Comment:
Why is the `institutionParameter` not part of the command?

Reply:
The institution was always used as a context. But was renamed to be the actor institution. The institution parameter is just one of the optional filter possibilities and is therefore not part of the factory method.